magpie/models.py

```diff
@@ -101,6 +101,15 @@ class GroupPermission(GroupPermissionMixin, Base):
     pass
 
 
+class UserGroupType(ExtendedEnum):
+    """
+    Supported types of user-group relationships.
+    """
+    ALL_USERGROUPS = "all_usergroups"
+    ACTIVE_USERGROUPS = "active_usergroups"
+    PENDING_USERGROUPS = "pending_usergroups"
```

Comment:
Remove redundant parts `_USERGROUPS` from names. 
Only `ALL = "all"`, etc. are sufficient. 
This will make requests easier to call/remember also:
```http
GET <>/users/me/groups?status=pending
GET <>/groups/people/users?status=all
```